### PR TITLE
fix add account issue

### LINKED ISSUE #1262: Adding new accounts with manual entry is broken
### Describe the issue

while doing manual entry it's not showing the code 

### Browser

Chrome

### Browser Version

127.0.6533.120

### Extension Version

8.0.0

### PR DESCRIPTION
**Pull request by Sneezry**
Fix https://github.com/Authenticator-Extension/Authenticator/issues/1262

### CODE REVIEW

File: src/models/otp.ts [hunk 1]
```diff
@@ -88,9 +88,21 @@ export class OTPEntry implements OTPEntryInterface {
           hash: string;
           index: number;
         },
-    encryption?: EncryptionInterface
+    encryption?: EncryptionInterface | Map<string, EncryptionInterface>
```
Comment (mymindstorm) [inline src/models/otp.ts:91]: This should just be a `Map<string, EncryptionInterface>`, the only place we need to have a case for `EncryptionInterface` is before migration logic runs.

File: src/models/otp.ts [hunk 2]
```diff
@@ -88,9 +88,21 @@ export class OTPEntry implements OTPEntryInterface {
...
   ) {
-    this.encryption = encryption;
+    if (encryption instanceof Map) {
+      for (const key in encryption) {
+        if (
+          /^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$/.test(
+            key
+          )
+        ) {
+          this.encryption = encryption.get(key);
```
Comment (mymindstorm) [inline src/models/otp.ts:100]: Use `state.accounts.defaultEncryption` instead of a loop

**Review by mymindstorm — CHANGES_REQUESTED** [review]
Please ignore previous comments.

Instead of making OTPEntry have multiple encryptions (we need to maintain a 1:1 relationship between an OTPEntry and an encryption), make 

https://github.com/Authenticator-Extension/Authenticator/blob/dev/src/components/Popup/AddAccountPage.vue#L131 

pass `state.accounts.defaultEncryption` instead.

**Comment by Sneezry** [comment]
Updated

**Review by mymindstorm — APPROVED** [review]

**Pull request merged**